main.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import FastAPI
+from database import engine, Base
+import crud
+from schemas import CityCreate, TemperatureCreate
+from database import get_db
+from fastapi import Depends
+from sqlalchemy.orm import Session
+
+# Ініціалізація бази даних
+Base.metadata.create_all(bind=engine)
+
+# Ініціалізація FastAPI
+app = FastAPI()
+
+
+# Ендпоінти для Cities
+@app.get("/cities/")
+def read_cities(db: Session = Depends(get_db)):
+    return crud.get_cities(db)
+
+
+@app.post("/cities/")
+def add_city(city: CityCreate, db: Session = Depends(get_db)):
+    return crud.create_city(db, city)

```

Comment:
The `add_city` endpoint is correctly implemented for adding a new city. However, the task description requires implementing additional endpoints such as updating and deleting cities. Consider adding these functionalities to fully meet the task requirements.